gtsfm/evaluation/metrics_report.py

```diff
@@ -179,25 +253,43 @@ def get_html_header() -> str:
               </head>"""
 
 
-def generate_metrics_report_html(metrics_groups: List[GtsfmMetricsGroup], html_path: str) -> None:
+def generate_metrics_report_html(
+    metrics_groups: List[GtsfmMetricsGroup],
+    html_path: str,
+    colmap_output_dir: str,
+    metric_paths: List[str],
+) -> None:
     """Generates a report for metrics groups with plots and tables and saves it to HTML.
 
     Args:
         metrics_groups: List of metrics to be reported.
+        html_path: Path where this report is written to (default: output_dir/gtsfm_metrics_report.html).
```

Comment:
there does not seem to be a default in the function, its just using whatever is passed in.

Reply:
The default referenced in this comment is the default argument for the method call in plot_metrics.py. The comment is now removed to avoid confusion.